ui/src/hosts/containers/HostPage.js

```diff
@@ -116,6 +117,14 @@ export const HostPage = React.createClass({
     );
   },
 
+  renderGraphTips() {
+    return (
+      `<p><code>Click + Drag</code> Zoom in (X or Y)</p>
+      <p><code>Shift + Click</code> Pan Graph Window</p>
+      <p><code>Double Click</code> Reset Graph Window</p>`
+    );
+  },
```

Comment:
I suppose I could throw it in the `data-tip` attribute, just seemed verbose as hell. This might be a bit excessive though